src/platform/plugins/shared/dashboard/server/usage/task_state/v1/schema.ts

```diff
@@ -28,14 +28,10 @@ export const versionSchema = schema.object({
     }),
     controls: schema.object({
       total: schema.number(),
-      chaining_system: schema.recordOf(schema.string(), schema.number()),
-      label_position: schema.recordOf(schema.string(), schema.number()),
-      ignore_settings: schema.recordOf(schema.string(), schema.number()),
       by_type: schema.recordOf(
```

Comment:
I'm no expert on Task Management schemas. Can we remove properties from task definitions without adding new task state version?